backend/src/bin/scripts/enrich-members-organizations.ts

```diff
@@ -0,0 +1,138 @@
+import commandLineArgs from 'command-line-args'
+import commandLineUsage from 'command-line-usage'
+import * as fs from 'fs'
+import path from 'path'
+import { getServiceLogger } from '@crowd/logging'
+import SequelizeRepository from '../../database/repositories/sequelizeRepository'
+import MemberRepository from '../../database/repositories/memberRepository'
+import { sendBulkEnrichMessage } from '../../serverless/utils/nodeWorkerSQS'
+import { ORGANIZATION_ENRICHMENT_CONFIG } from '../../conf'
+import OrganizationEnrichmentService from '../../services/premium/enrichment/organizationEnrichmentService'
+import OrganizationRepository from '../../database/repositories/organizationRepository'
+import getUserContext from '@/database/utils/getUserContext'
+
+/* eslint-disable no-console */
+
+const banner = fs.readFileSync(path.join(__dirname, 'banner.txt'), 'utf8')
+
+const log = getServiceLogger()
+
+const options = [
+  {
+    name: 'tenant',
+    alias: 't',
+    type: String,
+    description: 'The unique ID of tenant that you would like to enrich.',
+  },
+  {
+    name: 'help',
+    alias: 'h',
+    type: Boolean,
+    description: 'Print this usage guide.',
+  },
```

Comment:
let's also add two flags -o and -m so in the future we can enrich tenants selectively (ie: only enrich members or only enrich organizations) When both flags are present, or no flags given at all we can enrich both (like the default case)